### PR TITLE
Add GeForce NOW Presence

### LINKED ISSUE #1982: NVIDIA GeForce NOW | play.geforcenow.com
**Prerequisites and essential questions** 
- [x] Is it a popular site?
- [x] Is the website older than 2 months? 
- [ ] Is the site locked to a specific country/region?
- [ ] Is the site a paid service? (e.g. Netflix, Hulu)
**There are membership options, but its free**
- [x] Does the website feature NSFW content? (e.g. porn, etc...) 
**Depends on the game you’re playing**
- [ ] Are you a donator/patron?
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?**  christian#1111

**What's the name of the service?**  NVIDIA GeForce Now https://play.geforcenow.com/

**What should the Presence display?**  What game is the user playing and how long whenever they are playing it.

**If possible, please provide a logo for the service (512x512 minimum)**  https://i.ibb.co/xmdr1rG/73-F33769-93-CB-4-AD0-950-F-CD3-E4-D9-C4-F79.jpg

You need to follow these steps to access GFN on browser: https://www.reddit.com/r/GeForceNOW/comments/ichfaw/guide_play_gfn_play_on_any_computer_via_chrome/

### PR DESCRIPTION
**Pull request by Pepijn98**
<details>
<summary>Screenshots</summary>
<br>

browsing

game

playing
</details>

### CODE REVIEW

File: websites/G/GeForce NOW/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  },
+  "url": "play.geforcenow.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/a1L2Mi2.png",
+  "thumbnail": "https://i.imgur.com/JCyb4zk.png",
+  "color": "#76B918",
+  "tags": ["gaming", "streaming"],
+  "category": "games",
+  "iframe": false,
+  "readLogs": false
```
Comment (Bas950) [inline websites/G/GeForce NOW/dist/metadata.json:19]:
```suggestion
  "category": "games"
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
Add more to the presence, right now it's too low quality to be accepted.

**Comment by Pepijn98** [comment]
> Add more to the presence, right now it's too low quality to be accepted.

What else is there to add? You play games on GeForce NOW that's it.

**Comment by Bas950** [comment]
> > Add more to the presence, right now it's too low quality to be accepted.
> 
> What else is there to add? You play games on GeForce NOW that's it.

hmm yea I can't look at the website myself since I use edge which isn't supported xD

File: websites/G/GeForce NOW/presence.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  const privacy: boolean = await presence.getSetting("privacy"),
+    showTimestamp: boolean = await presence.getSetting("timestamp"),
+
+   presenceData: PresenceData = {
+    largeImageKey: "logo",
+    smallImageKey: "small"
+  };
+
+  if (showTimestamp) presenceData.startTimestamp = browsingStamp;
+  else delete presenceData.startTimestamp;
```
Comment (Bas950) [inline websites/G/GeForce NOW/presence.ts:18]:
```suggestion
    showTimestamp: boolean = await presence.getSetting("timestamp"),
    presenceData: PresenceData = {
      largeImageKey: "logo",
      smallImageKey: "small",
      startTimestamp: browsingStamp
    };

  if (!showTimestamp) delete presenceData.startTimestamp;
```

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**